src/components/Delete.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+export default function Delete(props) {
+    
+    // Function that delete Items from the list
+    const deleteItem = (id) => {
+        props.setItems((prevItem) => {
+            let newList = []
+            prevItem.forEach((item) => {
```

Comment:
Do this with [Array.filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter).